packages/block-library/src/details/editor.scss

```
@@ -0,0 +1,15 @@
+.wp-block-details {
+	.block-library-details__pseudo-summary {
+		margin-left: -1.25em;
+	}
+
+	.block-library-details__pseudo-summary::before {
+		content: "▶ ";
+	}
+}
```

To keep specificity low and easy to override, I'd simplify this to:
```suggestion

.block-library-details__pseudo-summary {
	margin-left: -1.25em;

	&::before {
		content: "▶ ";
	}
}
```

Also, is the `margin-left` really necessary?

---

Thanks, that looks better. 

I'm removing the `margin-left` although I do like the look of having the bullet outset and having the `RichText` vertically aligned with the blocks above and below. The problem with my approach is that ▶ and ▼ have different widths in some fonts, which makes it difficult to line up correctly. Maybe there's a CSS solution but I haven't found it yet.